Same index file on install + manifest update

sunny opened the pull request:
The `stimulus:manifest:update` task includes the `.js` extension, so the `stimulus:install` should include it as well to keep the starting file similar to the generated one.

marcoroth commented:
I think I would rather remove it from the manifest task then. 

Because it seems quite untypical to have the `.js` file ending in JavaScript imports

sunny commented:
Hey @marcoroth, thanks for the suggestion. I agree, that sounds much better. I have updated it to remove the `.js` file ending in the JS imports and added a test to ensure it supports `.ts` extensions as well.
